fields.php

```diff
@@ -349,3 +380,26 @@ function ( $taxonomy ) use ( $news_post_types ) {
 if ( function_exists( 'fm_register_submenu_page' ) ) {
 	fm_register_submenu_page( 'voicewp-settings', 'tools.php', __( 'Alexa Skill Settings', 'voicewp' ), __( 'Alexa Skill Settings', 'voicewp' ), 'manage_options', 'voicewp-settings' );
 }
+
+/**
+ * Creates option of user defined dictionary terms for replacement within
+ * Alexa content. Uses the 'sub' element to specify pronunciations of words.
+ *
+ * @param array $data FM data
+ * @return array
+ */
+function voicewp_fm_submenu_presave_data( $data ) {
+	if ( empty( $data['user_dictionary']['dictionary'] ) ) {
```

Comment:
Not a blocker. You might want to check that this is an array as well since you use it in a foreach below.